src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -47,6 +47,7 @@ import {
 } from "v2/Components/AddressForm"
 import { Router } from "found"
 import { Component } from "react"
+import { COUNTRIES_IN_EUROPEAN_UNION } from "@artsy/commerce_helpers"
```

Comment:
If you pull this branch down locally do you have any Type errors about this line?

It only started after I rebased on main
```
src/v2/Apps/Order/Routes/Shipping/index.tsx:50:45 - error TS2307: Cannot find module '@artsy/commerce_helpers' or its corresponding type declarations.

50 import { COUNTRIES_IN_EUROPEAN_UNION } from "@artsy/commerce_helpers"
                                               ~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in src/v2/Apps/Order/Routes/Shipping/index.tsx:50

error Command failed with exit code 2.
```

idk if it is my local env, I did try to reset my node modules + yarn stuff:
```
% rm -rf node_modules && yarn cache clean && yarn install
```

but cant seem to fix the type error, seems like CI + the test suite is fine 🤷🏻‍♀️ anything seem off here?

Reply:
I'm not seeing the error like that after running yarn install and refreshing the tab. I see it like this
<img width="1039" alt="Screenshot 2022-06-24 at 11 25 25" src="https://user-images.githubusercontent.com/7518671/175506410-4c876c0e-f2db-4181-875a-292781983f3c.png">
Not sure if this is something that is better addressed in the module or here in Force.